posthog/queries/paths/paths_actors.py

```diff
@@ -35,14 +36,12 @@ def actor_query(self, limit_actors: Optional[bool] = True) -> Tuple[str, Dict]:
         if self.should_query_funnel():
             paths_funnel_cte = self.get_path_query_funnel_cte(cast(Filter, self._funnel_filter))
 
-        select_statement = "DISTINCT person_id AS actor_id"
-        group_statement = ""
+        select_statement = "DISTINCT person_id AS actor_id, COUNT(*) as value_at_data_point"
+        group_statement = "GROUP BY actor_id"
         if self._filter.include_recordings:
-            select_statement = """
-                person_id AS actor_id
+            select_statement += """
                 , groupUniqArray(100)((timestamp, uuid, "$session_id", "$window_id")) as matching_events
             """
```

Comment:
If you rather want to be safe, and since we ultimately don't need this in the future, I would also be fine to just not change it here. What do you think?

Reply:
My fear is mostly that we'll lose time by accidentally breaking something we didn't know about. E.g. some odd untested combination of filters will stop working for some user... For trends we have cases like "formula" + "group property filter" + "cohort breakdown" = "world of hurt". I'm afraid of introducing more. So my vote goes for just adding this to the new HogQL version.